lib/internal/bootstrap/switches/is_main_thread.js

```diff
@@ -157,7 +157,11 @@ function getStdin() {
 
     case 'FILE':
       const fs = require('fs');
-      stdin = new fs.ReadStream(null, { fd: fd, autoClose: false });
+      stdin = new fs.ReadStream(null, {
+        fd: fd,
+        autoClose: false,
```

Comment:
Why auto close false?

Reply:
I don't know, but my guess is that with a regular file you still want to have an open `stdin` after `EOF`
@joyeecheung is the one to ask